modules/gathering-data-specific-features.adoc

```diff
@@ -1,11 +1,17 @@
 // Module included in the following assemblies:
 //
 // * virt/logging_events_monitoring/virt-collecting-virt-data.adoc
+// * support/gathering-cluster-data.adoc
 
 //This file contains UI elements and/or package names that need to be updated.
 
 // Dependency: modules/virt-document-attributes.adoc
 
+ifeval::["{context}" == "gathering-cluster-data"]
+:from-main-support-section:
+:VirtProductName: container-native virtualization
```

Comment:
FYI @aburdenthehand @ousleyp - sorry for the hack I had to do for this shared file, but just wanted to give you a heads up on this here. It's staying CNV right now, since this will go out with OCP 4.5 GA. But it should be changed to OpenShift virtualization whenever you guys are ready to go out w/ the name change.